packages/react-devtools-shared/src/backend/views/TraceUpdates/canvas.js

```diff
@@ -43,8 +43,30 @@ function drawNative(nodeToData: Map<HostInstance, Data>, agent: Agent) {
 }
 
 function drawWeb(nodeToData: Map<HostInstance, Data>) {
+  // if there are no nodes to draw, detach from top layer
+  if (nodeToData.size === 0) {
+    if (canvas !== null) {
+      try {
```

Comment:
Do you actually need to wrap it in try / catch now?

Reply:
Oh, I initially added that to solve Flow and type issues, but it became unnecessary after adding the annotations. I've removed it.